src/main/scala/me/rexim/morganey/Main.scala

```diff
@@ -54,9 +51,6 @@ object Main extends SignalHandler {
           con.println(e.getMessage)
           None
       }
-    } else {
-      con.println(nodeParseResult.toString)
-      None
```

Comment:
This is actually a bug. This `else` braunch printed syntax errors in REPL. After this change all the syntax errors are suppressed in REPL.